.gitignore

```diff
@@ -45,4 +46,4 @@ ctx.c
 *.ci8.png
 
 # Per-user configuration
-.python-version
+.python-version
```

Comment:
and if you can just add a new line back to the end of the file here, ty

Reply:
Done